nextjs-app/src/pages/form.tsx

```diff
@@ -67,42 +68,47 @@ export default function Form() {
                     onInput={handleUsernameInput as any}
                     type="text"></PieInput>
 
-                <label htmlFor="email">
+                <PieFormLabel for="email">
                     Email:
-                </label>
-                <input
+                </PieFormLabel>
+                <PieInput
```

Comment:
Please could we leave these input fields as native HTML? Only the first one should be using `PieInput` for now. 

I have a separate ticket to integrate the remaining types when they are ready to test (they aren't just yet)